src/deploy.rs

```diff
@@ -17,7 +17,7 @@ pub struct Cmd {
     #[clap(long, parse(from_os_str))]
     wasm: std::path::PathBuf,
     /// File to persist ledger state
-    #[clap(long, parse(from_os_str), default_value("ledger.json"))]
+    #[clap(long, parse(from_os_str), default_value(".soroban/ledger.json"))]
```

Comment:
Should this PR also update `.gitignore` to include `.soroban` like you mentioned in the PR description?